makefiles/in_chroot/install.sh

```
@@ -32,12 +34,17 @@ if ! [ -d /home/$USERNAME ]; then
 fi
 
 # add empty ~/.ssh/authorized_keys (see #80)
-SSH_AUTHORIZED_KEYS=/home/$USERNAME/.ssh/authorized_keys
-mkdir -p -m 700 $(dirname $SSH_AUTHORIZED_KEYS)
-chown $USERNAME:users $(dirname $SSH_AUTHORIZED_KEYS)
-touch $SSH_AUTHORIZED_KEYS
-chown $USERNAME:users $SSH_AUTHORIZED_KEYS
-chmod 600 $SSH_AUTHORIZED_KEYS
+function add_authorized_keys_file() {
+    SSH_AUTHORIZED_KEYS=/home/$1/.ssh/authorized_keys
+    mkdir -p -m 700 $(dirname $SSH_AUTHORIZED_KEYS)
```

Shellcheck (0.10.0)

[warning] When used with -p, -m only applies to the deepest directory.

(SC2174)